src/index.js

```diff
@@ -23,7 +23,7 @@
  *
  */
 
-import DatetimePicker from './components/DatetimePicker'
+import DatetimePicker from './components/DatetimePicker.vue'
```

Comment:
Below is a correct import syntax
`
import DatetimePicker from './components/DatetimePicker'
`